index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>@Sir_Malcom</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <section class="parent1">
+      <nav class="dcp1 navbar">
+          <img class="logo" src="./elements/lg.png" alt="Logo">
+          <div></div>
+          <img class="menu" src="./elements/Enabled.jpg" alt="Humbugger menu">
+      </nav>
+      <div class="space"></div>
+      <article class="dcp1 art1">
+          <h4>
+        I’m Sir_Malcom
+        Glad to see you!
+          </h4>
+      </article>
+      <article class="dcp1 art2">
+        I’m a software developer! I can help you build a product, 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.
+      </article>
+      <img class="dcp1 social" src="./elements/ltscnnct.jpg" alt="lets connect">
+      <div class="dcp1 parent2">
+        <div><img src="/elements/gthb.png" alt="github"></div>
+        <div><img src="/elements/lnkin.png" alt="linkedin"></div>
+        <div><img src="/elements/twtr.png" alt="twitter"></div>
+        <div><img src="/elements/m.png" alt="m"></div>
+        <div><img src="/elements/chill.png" alt="chill"></div>
+      </div>
```

Comment:
- Please make use of a `<ul>` here with the corresponding li with an `<a>` tag as those social icon media icons with be a link to your corresponding social pages. for example

```html
<ul>
  <li> <a href = "#" target = "_blank"> <img src="/elements/gthb.png" alt="github"> </a></li>
  <li> <a href = "#" target = "_blank"> <img src="/elements/lnkin.png" alt="linkedin"></a></li>
  <li> <a href = "#" target = "_blank"> <img src="/elements/twtr.png" alt="twitter"> </a></li>
  <li> <a href = "#" target = "_blank"><img src="/elements/m.png" alt="m"></a></li>
  <li> <a href = "#" target = "_blank"> <img src="/elements/chill.png" alt="chill"> </a></li>
</ul>
```
Then use CSS flex to style it appropriately. if you find this confusing, you can reach me on slack so I explain better `@kanu mike chibundu`